F1/F1/Constructor/ConstructorViewModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ConstructorViewModel.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/04/02.
+//
+
+import Foundation
+
+class ConstructorViewModel {
+
+    // MARK: Variables
+    private var repository: ConstructorRepositoryType?
+    private weak var delegate: ViewModelDelegate?
```

Comment:
```suggestion
     private weak var delegate: ViewModelDelegate?
    private var repository: ConstructorRepositoryType?
    private var constructorResults: [Race]?
```
Nitpick: Try keep variables of same type grouped together